bibtexparser/model.py

```diff
@@ -314,6 +314,10 @@ def pop_field(self, key: str) -> Optional[Field]:
         self._fields = [f for f in self._fields if f.key != key]
         return field
 
+    def __contains__(self, key: str) -> bool:
+        """Dict-mimicking ``in`` operator."""
+        return key in self.fields_dict
```

Comment:
This does involve constructing a `dict`; however, I did not find a directer way. If there is, it would be good to optimize (now or in the future)

Reply:
I am not overly concerned about the dict creation. While it is slow, this can be mitigated by#323 in a future PR. 

For the next pre-release, I guess we can live with the way it is now (as barely anyone will use this yet), and then we should make sure to address #323 before releasing the first stable version.